<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpecificSideIronPricingWindow</class>
 <widget class="QDialog" name="SpecificSideIronPricingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Material Prices</string>
  </property>
  <layout class="QVBoxLayout" name="addMachineVerticalLayout">
   <item>
    <widget class="QLabel" name="PricingTitleLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Side Iron Prices</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DynamicComboBox" name="sideIronComboBox">
     <property name="minimumSize">
      <size>
       <width>382</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>382</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="priceLabel">
       <property name="text">
        <string>Price</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="screwsLabel">
       <property name="text">
        <string>Screws</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="priceEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="screwsEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="acceptButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DynamicComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">widgets/DynamicComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>acceptButtons</sender>
   <signal>accepted()</signal>
   <receiver>SpecificSideIronPricingWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButtons</sender>
   <signal>rejected()</signal>
   <receiver>SpecificSideIronPricingWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
